src/authorize.js

```diff
@@ -37,16 +37,40 @@
       url += '&state=' + encodeURIComponent(redirectUri.substring(hashPos+1));
     }
     url += '&response_type=token';
+
+    if (global.cordova) {
```

Comment:
I've seen `window.cordova || window.PhoneGap || window.phonegap` used in some places. That's probably the safest option for legacy support.

Then again, https://github.com/nraboy/ng-cordova-oauth/blob/master/src/oauth.js uses just `window.cordova` and nobody has complained so far.

I'll leave the final decision to you guys.